### ISSUE TITLE
Add feature to have to TX in one Request

### ISSUE THREAD

**Issue by scroodge** [aliaksandr-aliakseyeu/Schedule#14]
Please implement as shown on picture
![image](https://github.com/Gollum959/Schedule/assets/1701571/763195e7-446a-4eb5-aba6-b23feeacea55)

**Comment by Gollum959** [comment]
Its notice for me. I have to think about creating additional table where I can connect some several requests in one request. It'll require minimal change to existing tables

**Comment by Gollum959** [comment]
Question for you. Do we clone all fields? Excluding city or PTS may move to another location?

**Comment by scroodge** [comment]
Yes - only difference is that in one day we have 2 TX start and Finish and also "ТРАКТ" times
